src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/annotatedregion/AnnotatedInterval.java

```diff
@@ -0,0 +1,88 @@
+package org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion;
+
+import com.google.common.collect.ImmutableSortedMap;
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.util.SortedMap;
+
+/**
+ * Simple class that just has an interval and sorted name-value pairs.
+ */
+public final class AnnotatedInterval implements Locatable  {
+
+    private SimpleInterval interval;
```

Comment:
It looks like this can be `final`

Reply:
@pshapiro4broad is here!  Now it's a party!  

Done.